### PR TITLE
Cherry pick #9341 to mrtk_dev

### PR DESCRIPTION
**Pull request by keveleigh**
## Overview

Brings over the contents of #9341 from @polar-kev

### CODE REVIEW

File: Documentation/GettingStartedWithMRTKAndXRSDK.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Getting started with MRTK and XR SDK
 
+> [!CAUTION]
+># We've moved! 
+>See this page on the ***[new docs website](https:docs.microsoft.com/windows/mixed-reality/mrtk-unity/configuration/getting-started-with-mrtk-and-xrsdk)***.
```
Comment (polar-kev) [inline Documentation/GettingStartedWithMRTKAndXRSDK.md:5]:
```suggestion
>See this page on the ***[new docs website](https://docs.microsoft.com/windows/mixed-reality/mrtk-unity/configuration/getting-started-with-mrtk-and-xrsdk)***.
```

**Review by polar-kev — CHANGES_REQUESTED** [review]

File: Documentation/GettingStartedWithTheMRTK.md [hunk 2]
```diff
@@ -8,6 +8,8 @@ ms.localizationpriority: low
 keywords: Unity,HoloLens, HoloLens 2, Mixed Reality, development, MRTK,
 ---
 
-# We've moved! 
-
-Check out our new [docs landing page](WelcomeToMRTK.md) to get started with MRTK in Unity!
\ No newline at end of file
+> [!CAUTION]
+># We've moved! 
+>We have a ***[new docs website](https://docs.microsoft.com/windows/mixed-reality/mrtk-unity/welcome-to-mrtk)*** for Mixed Reality Toolkit Documentation.
```
Comment (polar-kev) [inline Documentation/GettingStartedWithTheMRTK.md:13]:
```suggestion
>We have a ***[new docs website](https://docs.microsoft.com/windows/mixed-reality/mrtk-unity/)*** for Mixed Reality Toolkit Documentation.
```

File: Documentation/WelcomeToMRTK.md [hunk 3]
```diff
@@ -10,6 +10,12 @@ keywords: Unity,HoloLens, HoloLens 2, Mixed Reality, development, MRTK,
 
 # Welcome to MRTK
 
+> [!CAUTION]
+># We've moved! 
+>We have a ***[new docs website](https://docs.microsoft.com/windows/mixed-reality/mrtk-unity/welcome-to-mrtk)*** for Mixed Reality Toolkit Documentation.
```
Comment (polar-kev) [inline Documentation/WelcomeToMRTK.md:15]:
```suggestion
>We have a ***[new docs website](https://docs.microsoft.com/windows/mixed-reality/mrtk-unity)*** for Mixed Reality Toolkit Documentation.
```

File: README.md [hunk 4]
```diff
@@ -1,5 +1,11 @@
 ![Mixed Reality Toolkit](Documentation/Images/Logo_MRTK_Unity_Banner.png)
 
+> [!CAUTION]
+># We've moved! 
+>We have a ***[new docs website](https://docs.microsoft.com/windows/mixed-reality/mrtk-unity/welcome-to-mrtk)*** for Mixed Reality Toolkit Documentation.
```
Comment (polar-kev) [inline README.md:5]:
```suggestion
>We have a ***[new docs website](https://docs.microsoft.com/windows/mixed-reality/mrtk-unity)*** for Mixed Reality Toolkit Documentation.
```

**Review by polar-kev — CHANGES_REQUESTED** [review]

**Review by davidkline-ms — APPROVED** [review]

**Comment by keveleigh** [comment]
@polar-kev is this still needed and/or good to go?

**Comment by RogPodge** [comment]
Closing this PR since some other PR's have sufficiently covered what we were trying to do here: https://github.com/microsoft/MixedRealityToolkit-Unity/pull/9436

**Pull request closed without merging**